snap-gpf/src/main/java/org/esa/snap/core/gpf/internal/TileImpl.java

```diff
@@ -121,6 +121,17 @@ public double toGeoPhysical(double sample) {
         return rasterDataNode.scale(sample);
     }
 
+    private int toRaw(int sample) {
+        final double rawSample = rasterDataNode.scaleInverse(sample);
+        if (rawSample < -2147483648.0) {
```

Comment:
Which miracle Number ist this?
Why not
`   if (rawSample < Integer.MIN_VALUE)`